libs/datamodel/core/src/configuration/env_vars.rs

```diff
@@ -0,0 +1,25 @@
+use crate::ast;
+use serde::Serialize;
+
+#[serde(rename_all = "camelCase")]
+#[derive(Clone, Debug, Serialize, PartialEq)]
+pub struct StringFromEnvVar {
+    /// the name of the param this String / EnvVar is fore
+    #[serde(skip_serializing)]
+    pub name: &'static str,
+    /// contains the name of env var if the value was read from one
+    pub from_env_var: Option<String>,
+    pub value: String,
+}
+
+impl StringFromEnvVar {
+    pub fn to_arg(&self) -> ast::Argument {
```

Comment:
I am not so sure we should introduce a dependency from `configuration` to `ast` here. According to the architectural overview laid out in `lib.rs` this should not be the case.

Reply:
After reading the other code it seems more logical to me to put this function into the `dml_to_ast` module so that the serializers can use it.

Reply:
moved it.